basyx.tck/basyx.tck.AASServer/src/test/java/org/eclipse/basyx/testsuite/regression/aas/metamodel/AASAggregatorSuiteWithDefinedURL.java

```diff
@@ -183,6 +184,11 @@ private void checkSM(ISubmodel expected, ISubmodel actual) {
 			assertTrue(actual.getSubmodelElements().containsKey(id));
 		}
 	}
-
+	
+	private void assertEqualAsset(Asset expected, Asset actual) {
+		assertEquals(expected.get("modelType"), actual.get("modelType"));
```

Comment:
Please use string constants. They should be defined in a class somewhere

Reply:
I replaced it with ModelType enum value